crates/virtio-queue/src/lib.rs

```diff
@@ -190,4 +192,11 @@ pub trait QueueStateT: for<'a> QueueStateGuard<'a> {
 
     /// Set the index for the next descriptor in the used ring.
     fn set_next_used(&mut self, next_used: u16);
+
+    /// Pop and return the next available descriptor chain, or `None` when there are no more
```

Comment:
Can you also add some description about how this is suppose to be used (i.e. in respect to the already existing iterator; i think you mention this is an alternative way of consuming the iterator)?

Reply:
Added an extra phrase to the comment. Didn't mention the iterator, as it doesn't even exist strictly in the context of the `QueueStateT` trait.